components/tools/OmeroM/src/annotations/writeFileAnnotation.m

```diff
@@ -59,7 +63,8 @@
     originalFile.setMimetype(rstring(ip.Results.mimetype));
 end
 
-originalFile = updateOriginalFile(session, originalFile, filePath);
+originalFile = updateOriginalFile(...
+    session, originalFile, filePath, varargin{:});
```

Comment:
@joshmoore commit 15b29dcaec4f170e50f7bb42e5c920dde7382b8a should have fixed this issue.